melodic/distribution.yaml

```diff
@@ -13626,6 +13626,23 @@ repositories:
       type: git
       url: https://github.com/JdeRobot/VisualStates.git
       version: master
+  vl53l1x_ros:
+    doc:
+      type: git
+      url: https://github.com/okalachev/vl53l1x_ros.git
```

Comment:
The source licensing here seems...murky.  The top-level [LICENSE](https://github.com/okalachev/vl53l1x_ros/blob/master/LICENSE) file says MIT, the [package.xml](https://github.com/okalachev/vl53l1x_ros/blob/master/vl53l1x/package.xml) matches that, and the files in [src](https://github.com/okalachev/vl53l1x_ros/blob/master/vl53l1x/src/vl53l1x.cpp) also have that header.  That's all fine.

However, the files under [lib](https://github.com/okalachev/vl53l1x_ros/tree/master/vl53l1x/lib) are a different story.  Those seem to have a dual ST proprietary license and a BSD license.  I briefly read through the ST proprietary license, and while I am not a lawyer, it does seem to say that you cannot relicense this, nor combine it with other open-source software for redistribution.  So it's not at all clear to me that dual-licensing it with BSD is allowed.

@deadln Can you explain more about where the dual licensing of the files under lib/ comes from?

Reply:
Hi! This library is downloaded from the official ST website.

If it says one of the licenses available to that is BSD, then it's a free software, isn't it? So I can bundle it to my free software, why not?

If you say, that I must change the license of the package itself to BSD, I can do that.

Reply:
> If it says one of the licenses available to that is BSD, then it's a free software, isn't it? So I can bundle it to my free software, why not?

That's true, you definitely can if that is the case.  Can you provide the link to the ST site where you downloaded the code originally?

Reply:
I was a long time ago. I believe it could be this download: https://www.st.com/en/embedded-software/stsw-img007.html. But they have changed the library significantly since then.

Reply:
The reason I'm concerned is that in my previous dealings with ST, none of the code was open source.  Further, even just trying to download the code from the link above requires acceptance of a non-open-source license.  That doesn't mean that some of this code isn't available under the BSD license, but it does mean that it is hard for us to verify whether that is the case.

Reply:
@clalancette I'm also not a lawyer, but I think the header on each file, for instance https://github.com/okalachev/vl53l1x_ros/blob/master/vl53l1x/lib/core/vl53l1_api.c, is pretty clear - they dual license under either their old, terrible proprietary license, or under a 3-clause BSD, at YOUR option. ST has slowly been updating their licensing terms over the past few years, and so I think it's pretty clear the library is BSD.

Claiming the package is then MIT is a bit of a challenge - it'd be much easier if the package itself was 3-clause BSD.

Reply:
@clalancette, header in every source file in this library clearly states, that this library may be licensed with BSD license. And I didn't change them, of course.

Reply:
@clalancette the recent Noetic sync is out but our package does not appear in rosdistro. Should we change the overall licence in our source repo, or not?

Reply:
@clalancette We've managed to change overall licencing to BSD. Please confirm changes